src/Microsoft.DocAsCode.DataContracts.ManagedReference/LinkInfo.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.DataContracts.ManagedReference
+{
+    using System;
+    using YamlDotNet.Serialization;
+
+    using Microsoft.DocAsCode.DataContracts.Common;
+    using Microsoft.DocAsCode.Utility.EntityMergers;
+    using Newtonsoft.Json;
+
+    [Serializable]
+    public class LinkInfo
```

Comment:
seems exception only supports cref attribute, i think crefinfo is more appropriate?